src/ValueFormatters/QuantityFormatter.php

```diff
@@ -38,6 +38,12 @@ class QuantityFormatter extends ValueFormatterBase {
 	const OPT_APPLY_ROUNDING = 'applyRounding';
 
 	/**
+	 * Option key controlling whether the quantity's unit of measurement should be included
+	 * in the output.
+	 */
+	const OPT_APPLY_UNIT = 'applyUnit';
```

Comment:
Why should it be simple? A Quantity is a complex object consisting of a three decimal values and a unit. DecimalFormatter::format is simple.